locales/en/actions.json

```diff
@@ -41,12 +41,10 @@
   "save_and_return_to_overview": "Save and return to overview",
   "view_more_information": "View more information",
   "review": "Review",
-  "start_person_escort_record": "Start Person Escort Record",
+  "start_assessment": "Start $t({{context}})",
+  "print_assessment": "Print $t({{context}})",
   "provide_confirmation": "Provide confirmation",
   "confirm_and_complete_record": "Confirm and complete record",
-  "continue_person_escort_record": "Continue record",
-  "view_person_escort_record": "View Person Escort Record",
```

Comment:
Did this get replaced with a more generic context style message anywhere?

Reply:
These aren't actually used anywhere. They snuck into an old commit which ended up refactoring out the use of these keys.

So this is more of a tidy up.